eng/versioning/version_client.txt

```diff
@@ -3,26 +3,26 @@
 
 com.azure:azure-sdk-parent;1.5.0;1.5.0
 com.azure:azure-client-sdk-parent;1.6.0;1.6.0
-com.azure:azure-core;1.0.0;1.0.0
-com.azure:azure-core-amqp;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-http-netty;1.0.0;1.0.0
-com.azure:azure-core-http-okhttp;1.0.0;1.0.0
-com.azure:azure-core-management;1.0.0-preview.7;1.0.0-preview.7
-com.azure:azure-core-test;1.0.0;1.0.0
-com.azure:azure-data-appconfiguration;1.0.0-preview.6;1.0.0-preview.6
-com.azure:azure-identity;1.0.0;1.0.0
-com.azure:azure-security-keyvault-certificates;4.0.0-preview.5;4.0.0-preview.5
-com.azure:azure-security-keyvault-keys;4.0.0;4.0.0
-com.azure:azure-security-keyvault-secrets;4.0.0;4.0.0
-com.azure:azure-messaging-eventhubs;5.0.0-preview.5;5.0.0-preview.5
-com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.0-preview.3;1.0.0-preview.3
+com.azure:azure-core;1.1.0-preview.1;1.1.0-preview.1
```

Comment:
Ideally we wouldn't bump the dependency version here and keep it at GA. I release we still need to module references for now but if we can avoid bumping the dependency versions we should do so.